introduction.md

```diff
@@ -1,3 +1,5 @@
 My intro
 git
 JavaScript
+*Git
+*JavaScript
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?